From ae092fbf0b97b830862afc4ae81d168f869c8b9a Mon Sep 17 00:00:00 2001
From: Keir Fraser <keir.fraser@citrix.com>
Date: Fri, 4 Mar 2011 08:58:17 +0000
Subject: [PATCH 24/26] xen: events: Remove redundant clear of l2i at end of
 round-robin loop

Signed-off-by: Keir Fraser <keir.fraser@citrix.com>
Signed-off-by: Ian Campbell <ian.campbell@citrix.com>
[ijc: forward ported from linux-2.6.18-xen.hg 991:9ba6d9f3fbc0]
(cherry picked from commit acad13511ebe1db666aab5807117d3ac647ea58d)

Signed-off-by: Ian Campbell <ijc@hellion.org.uk>
---
 drivers/xen/events.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/xen/events.c b/drivers/xen/events.c
index 5a34789..9292d46 100644
--- a/drivers/xen/events.c
+++ b/drivers/xen/events.c
@@ -1148,7 +1148,6 @@ static void __xen_evtchn_do_upcall(struct pt_regs *regs)
 				pending_words &= ~(1UL << word_idx);
 
 			word_idx = (word_idx + 1) % BITS_PER_LONG;
-			bit_idx = 0;
 		}
 
 		BUG_ON(!irqs_disabled());
-- 
1.7.5.4

